docs/BacwardCompatability.md

```diff
@@ -0,0 +1,51 @@
+# Compatability Requirements and Implementation Plan
```

Comment:
typo in the name of the file

Reply:
yeep, thanks (I like to write this word with "a")